src/test/java/edu/harvard/iq/dataverse/api/DataverseFeaturedItemsIT.java

```diff
@@ -27,6 +29,94 @@ public static void setUpClass() {
         RestAssured.baseURI = UtilIT.getRestAssuredBaseUri();
     }
 
```

Comment:
For the new tests you’ve added, could you include some minor comments to describe and separate each test case? (Like we do in other tests) It would make it easier to follow each one. [Example](https://github.com/IQSS/dataverse/blob/develop/src/test/java/edu/harvard/iq/dataverse/api/DataverseFeaturedItemsIT.java#L36)